examples/forecasts/basic_example_forecasting.py

```diff
@@ -0,0 +1,54 @@
+from forecasting.utils.ml.models.mqcnn.mqcnn_model import MQCNNModel
+from forecasting.task.forecasting.dataset import TimeSeriesDataset
+import core as ag
+
+
+dataset = TimeSeriesDataset(
+    train_path="./COV19/processed_train.csv",
```

Comment:
Can you make this path a string variable that could be changed if we want to run other datasets?